### DISCUSSION TITLE
VectorStyle line with alpha set weird result

### DISCUSSION THREAD

**Discussion by Smurfj3** [Mapsui/Mapsui#2155] [Q&A] [answered]
Creating a vector style like so:

```
var vs = new VectorStyle
{
    Fill = null,
    Outline = null,
    Line = { Color = new Mapsui.Styles.Color(255, 255, 255, 100), Width = 4f }
};

```
Does not seem to work the way you'd think. I expect a white line that has the alpha set to 100, which should result in a slightly transparent white line, however the result is the expected white line and on top of it a thinner fully opaque black line that is not supposed to be there.

**Comment by charlenni** [comment]
Did you have a layer style too? If yes, delete this.

**Reply by Smurfj3** [reply]
No I have not set any layer style, I created a small example application just to show the problem. 

https://drive.google.com/file/d/1dYMLkFYI-DRkEity7LSKgvbmhn2uBEam/view?usp=sharing

I can't figure out how to get rid of the black line.

**Comment by Smurfj3** [comment, marked as answer]
Okay, so it was indeed the Layer Style that was causing it eventhough I didn't set a layer style. Apparently a default layer style is always set, so you have to explicitly set the layer style to null. Thanks to @charlenni for pointing me in the right direction :)

**Reply by pauldendulk** [reply]
It is a common issue, it is in our FAQ https://mapsui.com/documentation/faq.html#how-can-i-get-rid-of-the-white-dots-or-black-lines. The default was added to uses could at least see something when adding a feature. Since it is so confusing I guess it has to go. The change will result in some confusion too. Also, perhaps the Layer.Style should also become a list like the Feature.Styles.

We may also want to add a default style for everything which is intended for debug only.

**Reply by Smurfj3** [reply]
I completely understand why it is done this way and agree it would be confusing either way. I am not entirely sure if there is a less confusing way. I think I am also partly at fault for not reading the FAQ, I just glanced over the Quickstart and the Samples, because I noticed very early on there is not a lot of documentation, so I didn't even bother looking for the FAQ, which I should have.
